### PR TITLE
Update to 5.0

### PR DESCRIPTION
**Pull request by xoviat**
I don't have time to update the cling support and I don't think it should block others from receiving the newer LLVM.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by xoviat** [comment]
What does that do?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-{% set shortversion = "4.0" %}
-{% set version = "4.0.0" %}
-{% set sha256 = "8d10511df96e73b8ff9e7abbfb4d4d432edbdbe965f1f4f07afaf370b8a533be" %}
+{% set shortversion = "5.0" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "e35dcbae6084adcf4abb32514127c5eabd7d63b733852ccdb31e06f1373136da" %}
 
-{% set llvm_variant = os.environ.get('LLVM_VARIANT', 'default') %}
+{% set llvm_variant = "default" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you put revert this change? After the rerendering, you don't need these.

Comment (xoviat) [reply recipe/meta.yaml]: Will do.

**Comment by jakirkham** [comment]
Thoughts @SylvainCorlay?

**Comment by xoviat** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-{% set shortversion = "4.0" %}
-{% set version = "4.0.0" %}
-{% set sha256 = "8d10511df96e73b8ff9e7abbfb4d4d432edbdbe965f1f4f07afaf370b8a533be" %}
+{% set shortversion = "5.0" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "e35dcbae6084adcf4abb32514127c5eabd7d63b733852ccdb31e06f1373136da" %}
 
 {% set llvm_variant = os.environ.get('LLVM_VARIANT', 'default') %}
-{% set build_number = "0" %}
+{% set build_number = "1" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: Can you set this to 0?

Comment (xoviat) [reply recipe/meta.yaml]: must have been added in 82abe4c.

**Comment by xoviat** [comment]
@isuruf I resolved merge conflicts on the appveyor and circle files. They may be incorrect, hence the request for a rerender.

**Comment by isuruf** [comment]
Looks like it's okay because the bot would have pushed a commit if there was a change. I guess we should update the bot to comment that there was no change.

**Comment by isuruf** [comment]
Can someone cancel the older circle-ci builds?

**Comment by xoviat** [comment]
You guys should also disable the rolling appveyor builds.

**Comment by jakirkham** [comment]
Do you not have permissions @isuruf? If not, I think you should have them so this can proceed smoothly. Can you please send a PR adding yourself as a maintainer? Feel free to skip CI in it. Will merge right away.

**Comment by xoviat** [comment]
@jakirkham Do you know how to enable rolling builds on appveyor? I would highly recommend it.

**Comment by jakirkham** [comment]
We have a script that autocancels outdated builds when they start and AppVeyor has us on one of their nicer tiers. So builds actually start pretty fast after each other. Though I certainly can sympathize with rolling builds being more performant and desirable.

That said, one of the early creators of conda-forge had cited issues with rolling builds when he add the initial autocancellation script, but never got details as to what they were. The initial script had come from the Julia developers, who noted the same issues. They still use that script. So presumably these issues have gone unresolved.

Though we can still discuss enabling rolling builds, but am a little hesitant enabling them everywhere without proof that they do not cause problems. Issue ( https://github.com/conda-forge/conda-smithy/issues/582 ) is focused on Travis CI from the outset, but it seems like a good place to talk about other CIs as well.

**Comment by xoviat** [comment]
I guess the OSX builds may be done by next week (only half jokingly).

**Comment by isuruf** [comment]
Since the cling builds are disabled in `clangdev`, I guess this is okay.
